llc/llc-evolution.md

```diff
@@ -75,7 +75,7 @@ public virtual Response GetPetsByName(string petName, string sort, int? limit, i
 public virtual Response GetPetsByName(string petName, string sort = "ASC", int? limit = 20, int? skip = 0, string species = null, RequestOptions requestOptions = default);
 ```
 
-We can use this strategy to add new optional parameters to methods.  Note that since the parameter of a method is always of type `RequestOptions` we can not run into an issue where the new overload creates an ambiguity between different overloads.
+We can use this strategy to add new optional parameters to methods.  Note that since the last parameter of a method is always of type `RequestOptions` we cannot run into an issue where the new overload creates an ambiguity between different overloads.
```

Comment:
Is it possible to run into issues if the `RequestOptions` parameter has a default value?

Reply:
It's possible we could run into an issue here (especially for source breaking changes) by making these defaultable, especially now with the thinking around /not/ putting in a cancellation token parameter.  I need to think more about that.  I think that the strategy we employ does make us safe from introducing binary breaking changes.